docs/config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = "https://edge.docs.radapp.io/"
```

Comment:
given that you're targeting `edge`, i believe all the references in this file should remain as `edge`

Reply:
Updated